fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckContent.kt

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.BottomSheetHandle
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+private val bottomSheetShape = RoundedCornerShape(topStart = 16.dp, topEnd = 16.dp)
+private const val BOTTOM_SHEET_HANDLE_WIDTH_PERCENT = 0.1f
+
+/**
+ * A composable to display content for ReviewQualityCheck bottom sheet.
+ *
+ * @param onRequestDismiss Invoked when a user actions requests dismissal of the bottom sheet.
+ */
+@Composable
+fun ReviewQualityCheckContent(
```

Comment:
thoughts?
```suggestion
fun ReviewQualityCheck(
```

Reply:
Yeah the reason this is named like this is because I foresee a container that would observe the state from the store and this Content composable would get the data and functions and would eventually be the one used in previews. Would be interesting to see how it evolves, I'm not against the name change but it might get confusing if we do add a parent composable. wdyt?

Reply:
I think what you said makes sense! A lot of my nits are for this code in its current state and not how we'll long term iterate on it.